features/support/paths.rb

```diff
@@ -262,6 +262,12 @@ def path_to(page_name)
     when /^the [mM]y account page$/
       '/my/account'
 
+    when /^the [aA]ccess [tT]oken page$/
+      '/my/access_token'
+
+    when /^the [sS]ettings page$/
+      '/my/settings'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.